### ISSUE TITLE
Am getting error as invalid domain when I try to give .in domain

### ISSUE THREAD

**Issue by Nandhanrajvir** [D3Ext/AORT#5]
Am getting error as invalid domain when I try to give .in domain

**Comment by D3Ext** [comment]
Could you share the whole output?

**Comment by Nandhanrajvir** [comment]
When I tried with example.com am getting the flow and it is working fine.
But when I tried with my domain like for example: something.something.in it
is throwing an error message as invalid domain format.

**Comment by gab-ehcoud** [comment]
![image](https://user-images.githubusercontent.com/39666624/191251649-f81e6490-aad9-4dc1-b2a9-e9a77af203f5.png)

Working Fine in my system! 
Maybe try again one more time. It's throwing error in the first try.

**Comment by Nandhanrajvir** [comment]
Sure will try and update you.

**Comment by D3Ext** [comment]
The main purpose of the tool is to launch the recon against a domain like `example.com` but not like `example.gov.in` (note the 3 parts seperated by dots) however I will try it and fix it in case of errors

**Comment by akisakye** [comment]
@D3Ext am I reading ok that this is targeting only .com TLD? I had it fail for .com.au

```
$ aort -d example.com.au

Invalid domain format, example: domain.com
```

**Comment by D3Ext** [comment]
Sorry, let me change one thing, and it should work

**Comment by D3Ext** [comment]
Now it should work

**Comment by akisakye** [comment]
Is the new version available with pip?

**Comment by D3Ext** [comment]
No, I quickly fixed it before going to sleep, today I upload it as package

**Comment by Nandhanrajvir** [comment]
I tried with example.example.exam.in but it shows invalid domain format

**Comment by D3Ext** [comment]
@Nandhanrajvir use the latest version directly from github, I'll upload it now as package

**Comment by Nandhanrajvir** [comment]
Sure will try and update

**Comment by rudSarkar** [comment]
Hey @D3Ext ,

the issue has been fixed on the commit 16aed56fcd53db4ebbee65e99eeaa2f3391485eb I have checked by myself, I have a suggestion you can set regex to validate the domain name as I have seen you’ve comment out the validation process.

**Comment by D3Ext** [comment]
Ok, I will change that , thanks for your suggestion :)
